### PR TITLE
Create user command, server + client handlers, tests

### PR DESCRIPTION
**Pull request by spetz**

### CODE REVIEW

File: iggy/src/users/create_user.rs [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+        assert_eq!(command.status, status);
+        assert!(command.permissions.is_none());
+    }
+
+    #[test]
+    fn should_be_read_from_string() {
+        let username = "user";
+        let password = "secret";
+        let status = UserStatus::Active;
+        let input = format!("{}|{}|{}", username, password, status);
```
Comment (BartoszCiesla) [inline iggy/src/users/create_user.rs]: use `{username}|{password}|{status}`

File: server/src/binary/handlers/users/create_user_handler.rs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+use tokio::sync::RwLock;
+use tracing::trace;
+
+pub async fn handle(
+    command: &CreateUser,
+    sender: &mut dyn Sender,
+    user_context: &mut UserContext,
+    system: Arc<RwLock<System>>,
+) -> Result<(), Error> {
+    trace!("{}", command);
```
Comment (BartoszCiesla) [inline server/src/binary/handlers/users/create_user_handler.rs]: `trace!("{command}")`

**Review by BartoszCiesla — APPROVED** [review]

File: iggy/src/users/create_user.rs (hunk 1 above)
Comment (spetz) [reply iggy/src/users/create_user.rs]: Good point :)

File: server/src/binary/handlers/users/create_user_handler.rs (hunk 2 above)
Comment (spetz) [reply server/src/binary/handlers/users/create_user_handler.rs]: Yeah, again a copypasta :D

**Pull request merged**